docs/application/web/guides/cordova/globalization.md

```diff
@@ -16,18 +10,18 @@ The main features of the Globalization API include:
 
   You can obtain the following locale information:
 
-  - [Get the user's current language](./cordova/globalization-w.md#current_language) as the BCP 47 identifier (such as en-US).
-  - [Get the user's current locale](./cordova/globalization-w.md#current_locale) as the BCP 47 identifier (such as en-US).
-  - [Get the pattern string to format and parse currency](./cordova/globalization-w.md#currency) and the ISO 4217 currency code.
-  - [Get the names of months and names of the days of the week](./cordova/globalization-w.md#names_of_months_and_days).
-  - [Get the pattern string to format and parse dates](./cordova/globalization-w.md#date_format).
-  - [Get the first day of the week](./cordova/globalization-w.md#first_day_of_week).
-  - [Get the pattern string to format and parse numbers](./cordova/globalization-w.md#number_format).
-  - [Determine whether the Daylight Saving Time is in effect](./cordova/globalization-w.md#dst) for a given date.
+  - [Get the user's current language](#retrieving-the-current-kanguage) as the BCP 47 identifier (such as en-US).
```

Comment:
It looks like not `#retrieving-the-current-kanguage` but `#retrieving-the-current-language`

Reply:
done